### PR TITLE
source-scryfall contribution from michel-tricot

### PR DESCRIPTION
**Pull request by michel-tricot**
Auto-generated PR by the Connector Builder for source-scryfall

### CODE REVIEW

File: airbyte-integrations/connectors/source-scryfall/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Scryfall
+This directory contains the manifest-only connector for `source-scryfall`.
+
+For Magic The Gathering fans. Here is a simple data source for all the cards and sets!
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-scryfall/README.md:4]: Haha, good one! You still play, @michel-tricot?

File: airbyte-integrations/connectors/source-scryfall/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "*"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-scryfall/metadata.yaml:5]:
```suggestion
      - api.scryfall.com
```

File: airbyte-integrations/connectors/source-scryfall/README.md (hunk 1 above)
Comment (michel-tricot) [reply airbyte-integrations/connectors/source-scryfall/README.md:4]: Not really, used to 20y ago and introduced my son to it a few months ago :)

**Pull request merged**